presentation/src/main/java/com/anytypeio/anytype/presentation/editor/EditorViewModel.kt

```diff
@@ -990,9 +994,16 @@ class EditorViewModel(
         }
 
         jobs += viewModelScope.launch {
+            val networkMode = getNetworkMode.run(Unit).networkMode
```

Comment:
please make sure it's not running on the main thread.

Reply:
it's not